migrations/addresses.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Shall this file be committed here like this?

Reply:
Will actually be read, and written, by migration script, as to simplify both the deployment of new price feed examples, and the upgrading of anyone previously deployed. So yes, as part of this PR it could either be removed of left empy (i.e. "{}")